fastlane/Fastfile

```diff
@@ -234,6 +234,43 @@ lane :run_pod_install do |options|
   end
 end
 
+desc "Run automatic bump in other repos"
+lane :bump_hybrid_dependencies do |options|
+  bump = current_version_number
+  repo_name = options[:repo_name],
+
+  UI.message("Triggering pipeline to bump #{repo_name} dependency to #{bump}")
+
+  require 'net/http'
+  require 'uri'
+  require 'json'
+
+  uri = URI.parse("https://circleci.com/api/v2/project/github/RevenueCat/purchases-flutter/pipeline")
+  request = Net::HTTP::Post.new(uri)
+  request.basic_auth(ENV['CIRCLECI_TOKEN'], "")
+  request.content_type = "application/json"
+  request.body = JSON.dump({
+    "branch" => "main",
+    "parameters" => {
+      "action" => "upgrade-hybrid-common",
+      "hybrid-common-version" => "#{bump}",
+    }
+  })
+
+  req_options = {
+    use_ssl: uri.scheme == "https",
+  }
+
+  response = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
+    http.request(request)
+  end
+
+  puts response.code
+  puts response.body
```

Comment:
Is this debugging code? If it isn't, we should make sure no hidden information shows up in the logs in CircleCI.

Reply:
<img width="858" alt="Screen Shot 2022-09-27 at 1 14 41 PM" src="https://user-images.githubusercontent.com/664544/192511140-972c1017-09d8-48f6-b0be-d9bdc35b360d.png">

I left it to print the response from GitHub. Do you think we should remove it just in case?

Reply:
Hmm I'm not a fan of printing the whole response, specially since our test logs are open... I guess I would say to only print the response on errors maybe? And if we can parse the error message, instead of printing the whole response that would be even better.